src/main/java/com/meilisearch/sdk/KeysHandler.java

```diff
@@ -63,11 +76,10 @@ Key createKey(Key options) throws MeilisearchException {
      * @param key String containing the key
      * @param options String containing the options of the key
      * @return Key Instance
-     * @throws Exception if client request causes an error
+     * @throws MeilisearchException if client request causes an error
      */
-    Key updateKey(String key, Key options) throws Exception {
-        String urlPath = "/keys/" + key;
-        return httpClient.patch(urlPath, options, Key.class);
+    Key updateKey(String key, KeyUpdate options) throws MeilisearchException {
+        return httpClient.patch(new KeysQuery().toQuery(key), options, Key.class);
```

Comment:
Same as for `getIndex` and `getIndexes`, should `updateKey` and `updatePrimaryKey` somehow work in similar ways?